gui/landing/landing.html

```diff
@@ -44,7 +44,7 @@ <h3 class="panel-title">Already have an account?</h3>
         </div>
         <div class="panel-body">
           <p>Start using {{ productName }}!</p>
-          <a id="signInBtn" class="btn btn-default" ng-href="/login/{{ rd ? '?rd=' + rd : '' }}">Sign in here <i class="fa fa-fw fa-caret-right"></i></a>
+          <a id="signInBtn" class="btn btn-default" ng-href="{{ signinLink }}">Sign in here <i class="fa fa-fw fa-caret-right"></i></a>
```

Comment:
Hide the "Sign Up" button if OAuth is enabled, set "Sign In" target based on `SIGNIN_URL` envvar